skills_DS/base/models.py

```diff
@@ -54,7 +54,7 @@ class Profile(models.Model):
     user = models.OneToOneField(User,on_delete=models.CASCADE, related_name="profile")
     age = models.IntegerField(null=True)
     gender = models.CharField(max_length=20)
-    yearOfStudy = models.IntegerField(null=True)
+    yearOfStudy = models.CharField(max_length=2)
```

Comment:
Should be `IntegerField` instead of `CharField`. I don't think a length/digit limit is necessary.
Alternatively, you can use enumeration types as well.

Reply:
it's charfield as we have a value of "5+"

Reply:
Fair enough. I think we should use enumeration types in this case, since the options are limited. (You only have 5 choices to choose from)